frontend/pages/hosts/details/OSSettingsModal/OSSettingsTable/OSSettingsTableConfig.tsx

```diff
@@ -53,6 +54,42 @@ interface IDataColumn {
   sortType?: string;
 }
 
+/**
+ * generates the formatted tooltip for the error column.
+ * the expected format of the error string is:
+ * "key1: value1, key2: value2, key3: value3"
+ */
+const generateFormattedTooltip = (detail: string) => {
+  const keyValuePairs = detail.split(/, */);
```

Comment:
sorry for beaking this, seems like when the component is mounting `detail` is empty for a second and causes this function to throw an error and break the page.

just as an example, something like this fixed the issue, but I don't know what are our good practices to deal with this scenario.

```
  if (!detail) {
    return "";
  }
```

Reply:
you're right, we fixed it with @ghernandez345